index.html

```diff
@@ -7663,64 +7659,276 @@ <h2>Acknowledgements</h2>
 70RSAT21T00000016/P00001, 70RSAT23T00000005, 70RSAT23C00000030,
-70RSAT23R00000006, 70RSAT24T00000014, 70RSAT22T00000001, and the National Science
-Foundation under NSF 22-572. The content of this specification does not
+70RSAT23R00000006, 70RSAT24T00000014, 70RSAT22T00000001, and the National
+Science Foundation under NSF 22-572. The content of this specification does not
 necessarily reflect the position or the policy of the U.S. Government and no
 official endorsement should be inferred.
       </p>
 
       <p>
 The Working Group would like to thank the following individuals for reviewing
-and providing feedback on the specification (in alphabetical order):
+and providing feedback on the specification (in alphabetical order by first
+name):
       </p>
 
       <p>
-Christopher Allen, David Ammouial, Joe Andrieu, Bohdan Andriyiv, Ganesh
-Annan, Kazuyuki Ashimura, Tim Bouma, Pelle Braendgaard, Dan Brickley,
-Allen Brown, Jeff Burdges, Daniel Burnett, ckennedy422, David Chadwick,
-Chaoxinhu, Kim (Hamilton) Duffy, Lautaro Dragan, enuoCM, Ken Ebert, Eric
-Elliott, William Entriken, David Ezell, Nathan George, Reto Gmür, Ryan
-Grant, glauserr, Adrian Gropper, Joel Gustafson, Amy Guy, Lovesh
-Harchandani, Daniel Hardman, Dominique Hazael-Massieux, Jonathan Holt,
-David Hyland-Wood, Iso5786, Renato Iannella, Richard Ishida, Ian Jacobs,
-Anil John, Tom Jones, Rieks Joosten, Gregg Kellogg, Kevin, Eric Korb,
-David I. Lehn, Michael Lodder, Dave Longley, Christian Lundkvist, Jim
-Masloski, Pat McBennett, Adam C. Migus, Liam Missin, Alexander Mühle,
-Anthony Nadalin, Clare Nelson, Mircea Nistor, Grant Noble, Darrell
-O'Donnell, Nate Otto, Matt Peterson, Addison Phillips, Eric Prud'hommeaux,
-Liam Quin, Rajesh Rathnam, Drummond Reed, Yancy Ribbens, Justin Richer,
-Evstifeev Roman, RorschachRev, Steven Rowat, Pete Rowley, Markus
-Sabadello, Kristijan Sedlak, Tzviya Seigman, Reza Soltani, Manu Sporny,
-Orie Steele, Matt Stone, Oliver Terbu, Ted Thibodeau Jr, John Tibbetts,
-Mike Varley, Richard Varn, Heather Vescent, Christopher Lemmer Webber,
-Benjamin Young, Kaliya Young, Dmitri Zagidulin, and Brent Zundel.
+Abhishek Mahadevan Raju,
+Adam C. Migus,
+Adam Migus,
+Addison Phillips,
+Adrian Gropper,
+Aisp-GitHub,
+Alen Horvat,
+Alexander Mühle,
+AlexAndrei98,
+Allen Brown,
+Amy Guy,
+Andor Kesselman,
+Andres Paglayan,
+Andres Uribe,
+Andrew Hughes,
+Andrew Jones,
+Andrew Whitehead,
+Andy Miller,
+Anil John,
+Anthony Camilleri,
+Anthony Nadalin,
+ashimura,
+Benjamin Collins,
+Benjamin Goering,
+Benjamin Young,
+Bert Van Nuffelen,
+Bohdan Andriyiv,
+Brent Zundel,
+Brian Richter,
+Bruno Zimmermann,
+Bumblefudge,
+Butters,
+caribouW3,
+cdr,
+Chaoxinhu,
+Charles "Chaals" McCathieNevile,
+Charles E. Lehner,
+Chris Abernethy,
+Chris Buchanan,
+Christian Lundkvist,
+Christine Lemmer-Webber,
+Christoph Lange,
+Christopher Allen,
+Christopher Lemmer Webber,
+ckennedy422,
+Clare Nelson,
+confiks,
```

Comment:
```suggestion
```

Ditto about the proposed text in the acknowledgements not identifying the contributor.